submitted_models/ctu_cras_norlab_x500_sensor_config_1/model.sdf

```diff
@@ -0,0 +1,911 @@
+            <uri>meshes/holders/side_ctu.dae</uri>
+          </mesh>
+        </geometry>
+      </visual>
+      <visual name='base_link_fixed_joint_lump__os_sensor_link_visual_21'>
+        <pose>0 0 0.06 0 -0 1.5708</pose>
+        <geometry>
+          <mesh>
+            <scale>1 1 1</scale>
+            <uri>meshes/sensors/os1_64.dae</uri>
+          </mesh>
+        </geometry>
+      </visual>
+      <visual name='base_link_fixed_joint_lump__os_sensor_window_link_visual_22'>
+        <pose>0 0 0.0944 0 -0 0</pose>
+        <geometry>
+          <cylinder>
+            <length>0.035</length>
+            <radius>0.038</radius>
+          </cylinder>
+        </geometry>
+      </visual>
+      <visual name='base_link_fixed_joint_lump__pc_mount_link_visual_23'>
+        <pose>0 0 0 0 -0 0</pose>
+        <geometry>
+          <mesh>
+            <scale>1 1 1</scale>
+            <uri>meshes/holybro/holybro_x500_pc_mount.dae</uri>
+          </mesh>
+        </geometry>
+      </visual>
+      <visual name='base_link_fixed_joint_lump__pixhawk_link_visual_24'>
+        <pose>0 0 0.036 0 -0 0</pose>
+        <geometry>
+          <mesh>
+            <scale>1 1 1</scale>
+            <uri>meshes/sensors/pixhawk.dae</uri>
+          </mesh>
+        </geometry>
+      </visual>
+      <visual name='base_link_fixed_joint_lump__rear_left_leg_clip_link_visual_25'>
+        <pose>-0.162635 0.162635 -0.157 0 -0 2.35619</pose>
+        <geometry>
+          <mesh>
+            <scale>1 1 1</scale>
+            <uri>meshes/holybro/holybro_x500_leg.dae</uri>
+          </mesh>
+        </geometry>
+      </visual>
+      <visual name='base_link_fixed_joint_lump__rear_right_leg_clip_link_visual_26'>
+        <pose>-0.162635 -0.162635 -0.157 0 0 -2.35619</pose>
+        <geometry>
+          <mesh>
+            <scale>1 1 1</scale>
+            <uri>meshes/holybro/holybro_x500_leg.dae</uri>
+          </mesh>
+        </geometry>
+      </visual>
+      <visual name='base_link_fixed_joint_lump__right_cover_link_visual_27'>
+        <pose>0 -0.064 0.03 0 -0 0</pose>
+        <geometry>
+          <mesh>
+            <scale>1 1 1</scale>
+            <uri>meshes/holders/side_ctu.dae</uri>
+          </mesh>
+        </geometry>
+      </visual>
+      <visual name='base_link_fixed_joint_lump__rs_up_holder_link_visual_28'>
+        <pose>0.085 0 0.032 -1.5708 -0 -1.5708</pose>
+        <geometry>
+          <mesh>
+            <scale>1 1 1</scale>
+            <uri>meshes/holders/holder_rs_up.dae</uri>
+          </mesh>
+        </geometry>
+      </visual>
+      <visual name='base_link_fixed_joint_lump__sensor_holder_link_visual_29'>
+        <pose>0 0 -0.022 1.5708 0 -1.5708</pose>
+        <geometry>
+          <mesh>
+            <scale>1 1 1</scale>
+            <uri>meshes/holders/holder_rs_down_bfox.dae</uri>
+          </mesh>
+        </geometry>
+      </visual>
+    </link>
+    <joint name='imu_joint' type='revolute'>
```

Comment:
I think this should be fixed joint (change in URDF, too). These zero-motion revolute joints can be a source of instability.